### DISCUSSION TITLE
Weight Initialization

### DISCUSSION THREAD

**Discussion by gdemirezen** [pyg-team/pytorch_geometric#6390] [General]
Hello,
How can we initialize weights and biases of GNN operators like GraphConv, GATConv etc... in PyG?

**Comment by rusty1s** [comment]
Just as you would do in PyTorch. For example, if you want to initialize the weights of `GCNConv`, you can look up its code, see that is just uses a single `Linear` layer, and modify the weights of it, e.g., 
```python
torch.nn.init.xavier_uniform(conv.lin.weight)
```

**Reply by gdemirezen** [reply]
Thank you for your quick reply!
